libraries/chain/apply_context.cpp

```diff
@@ -250,6 +252,12 @@ void apply_context::require_recipient( account_name recipient ) {
          recipient,
          schedule_action( action_ordinal, recipient, false )
       );
+
+      if (auto dm_logger = control.get_deep_mind_logger()) {
+         fc_dlog(*dm_logger, "CREATION_OP NOTIFY ${action_id}",
```

Comment:
@arhag Your thoughts on this?

Reply:
This is what they had before. I suggested to move to the pointer version so that each controller can be configured to not due logging (more modular) and also so that it is more efficient (no lookup by string in the critical path).